pypoptools/pypoptesting/framework/generate_util.py

```diff
@@ -11,7 +11,7 @@
 def generate_endorsed(node: Node, mock_miner,
                       atvs: int, vtbs: int,
                       alt_blocks: int, vbk_blocks: int, btc_blocks: int,
-                      seed: int, timeout=60) -> None:
+                      seed: int, timeout=6000) -> None:
```

Comment:
don't set it to be too big, otherwise it would not be possible to distinguish between slow/failing code